documentation/pom.xml

```diff
@@ -22,202 +22,219 @@
     <description>The Hibernate Search reference manual</description>
 
     <properties>
+        <asciidoctor.base-outputdir>${basedir}/target/asciidoctor/en-US</asciidoctor.base-outputdir>
```

Comment:
Also, about the value of this property, didn't you say you were going to remove the `/en-US` part in because we don't translate the doc anyway? For consistency?